OloEngine/src/OloEngine/Cinematic/CinematicPlayer.cpp

```
@@ -0,0 +1,115 @@
+        std::stable_sort(hits.begin(), hits.end(),
+                         [](const auto& a, const auto& b)
+                         { return a.first < b.first; });
+
+        out.reserve(out.size() + hits.size());
+        for (const auto& [time, name] : hits)
+        {
+            out.push_back(*name);
+        }
+    }
+
+    TickResult Tick(const CinematicSequence& sequence, f32 fromTime, f32 eventFloor, f32 dt, f32 speed, bool loop)
```

Cppcheck (2.20.0)

[style] The function 'Tick' is never used.

(unusedFunction)